File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.js
function solution(n) {
    var answer = '';
    let s = '수';
    let b = '박';
    while(true){
        answer += s;
        if(answer.length == n)break;
        answer += b;
        if(answer.length == n)break;

    }
    return answer;
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.java
public class Solution {
    public String solution(int n) {
        String answer = "";
        answer += "수박".repeat(n/2) + (n % 2 == 1 ? "수":"");
        return answer;
    }

    public static void main(String[] args) {
        Solution sol = new Solution();
        int n = 3;
        System.out.println(sol.solution(n));
    }
}
